docs/management_api.yml

```diff
@@ -433,6 +532,51 @@ definitions:
       id: "806603def19d417d004a4b67e"
       created_ts: "2020-07-06T15:04:49.114046203+02:00"
       updated_ts: "2020-07-07T01:04:49.114046203+02:00"
+  PersonalAccessTokenRequest:
+    description: Personal Access Token Request.
+    type: object
+    properties:
+      name:
+        description: Name of a token.
+        type: string
+      expires_in:
+        description: Expiration time in seconds (maximum one year - 31536000s).
+        type: number
+    required:
+      - name
+      - expires_in
+    example:
+      name: 'my_personal_token'
+      expires_in: 28800
+  PersonalAccessToken:
+    description: Personal Access Token Object.
+    type: object
+    properties:
+      id:
+        description: Token identifier.
+        type: string
+      name:
+        description: Name of a token.
+        type: string
+      last_used:
+        description: |
+            Date of last usage of a token. The accuracy is 5 minutes.
+        type: string
+        format: date-time
+      expiration_date:
+        description: Expiration date.
+        type: string
+        format: date-time
+    required:
+      - id
+      - name
+      - last_used
+      - expiration_date
+    example:
+      id: "806603def19d417d004a4b67e"
+      name: 'my_personal_token'
+      last_used: '5 Jul 2022 11:11:35 GMT'
+      expiration_date: '16 Oct 2022 07:28:34 GMT'
```

Comment:
is there any particular reason to have this format? I'm sure this will be parsed without a problem, but I think there are ISO strings (like [here](https://github.com/mendersoftware/inventory/blob/master/docs/management_api.yml#L764)) in most places

Reply:
yes and I think this, in fact, is what backend is returning (but I'll double check that)